code/main.py

```diff
@@ -173,6 +173,24 @@ def main():
         print(f"::set-output name=run_metrics::{run_metrics}")
         print(f"::set-output name=run_metrics_markdown::{run_metrics_markdown}")
 
+        # Download artifacts if enabled
```

Comment:
1. Only download for successful runs
Good question. Maybe the user wants to download logs in case of an error?

2. Separate action
We are adding this functionality to allow the user to only train models on Azure and then do the deployment elsewhere. That's why I added it to the run action, but we could also separate it. At the same time, I am not aware of any other logs that the user may want to download. Specific log files are only created in case of a run.

3. Only log files
I do not have an opinion, whether log files should be included. Including log files would allow us to write them back to a PR. This might be useful in some cases.

@awmatheson @ashishonce Let's find a good solution for those questions.

Reply:
@ashishonce @marvinbuss 

1. Good point, we have problems we face with training models where it is sometimes difficult to view the logs in a single interface and the cost of switching between interfaces is quite high. Perhaps a dump of the logs to an artifact would be useful, but I am not sure if this is something we need to develop now. 

2. I am of the opinion that the artifacts being downloaded should not be a separate action, but rather a flag in the individual actions. Downloading at any point in the workflow seems like something that a user can build themselves if required.

Reply:
This should be final now. Feel free to propose additional changes.